web/src/components/Section.js

```diff
@@ -37,12 +38,14 @@ const Section = ({ children, id, resource }) => {
 Section.propTypes = {
   children: PropTypes.node.isRequired,
   id: PropTypes.string,
-  resource: PropTypes.string
+  resource: PropTypes.string,
+  sectionClass: PropTypes.string
```

Comment:
What do you think about making this a bool since there's only two values: empty and `component-section`?  Then you could turn it on with something like `<Section component>` and all the various places its used don't need to know what the class name is.  I'm also wondering if this should default to the `component-section` value since it's used in all but one place.

Reply:
ooooh, that makes sense to me!